debril/rss-atom-bundle/4.0/packages/rss_atom.yaml

```diff
@@ -0,0 +1,5 @@
+debril_rss_atom:
```

Comment:
you still haven't moved this file inside the `etc` folder, so it does not get copied